src/parser/jobs/mnk/modules/evaluators/PlayersBuffedEvaluator.tsx

```diff
@@ -0,0 +1,87 @@
+import {Trans} from '@lingui/react'
+import {StatusLink} from 'components/ui/DbLink'
+import {Status} from 'data/STATUSES'
+import {EvaluatedAction, EvaluationOutput, WindowEvaluator} from 'parser/core/modules/ActionWindow'
+import {HistoryEntry} from 'parser/core/modules/ActionWindow/History'
+import {SEVERITY, SeverityTiers, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+const DEFAULT_EXPECTED_PLAYER_COUNT = 8
+const DEFAULT_PLAYERS_MISSED_SEVERITY = {
+	1: SEVERITY.MINOR,
+}
+
+export interface PlayersBuffedEvaluatorOptions {
+	/**
+	 * This is used to determine how many players were buffed in a given window.
+	 * @param window The window to determine how many players were buffed
+	 * @returns The amount of players buffed in the window
+	 */
+	affectedPlayers: (window: HistoryEntry<EvaluatedAction[]>) => number
+	/**
+	 * Defaults to full party size.
+	 */
+	expectedPlayerCount?: number
+	/**
+	 * Defaults to minor.
+	 */
+	severityTiers?: SeverityTiers,
+	/**
+	 * The status of the buff players are receiving.
+	 */
+	status: Status['id'],
+	suggestionContent: JSX.Element,
+	suggestionIcon: string
+}
+
+export class PlayersBuffedEvaluator implements WindowEvaluator {
+	private readonly affectedPlayers: (window: HistoryEntry<EvaluatedAction[]>) => number
+	private readonly expectedPlayerCount: number;
+	private readonly severityTiers: SeverityTiers;
+	private readonly status: Status['id'];
+	private readonly suggestionContent: JSX.Element;
+	private readonly suggestionIcon: string;
+
+	constructor(opts: PlayersBuffedEvaluatorOptions) {
+		this.affectedPlayers = opts.affectedPlayers
+		this.expectedPlayerCount = opts.expectedPlayerCount || DEFAULT_EXPECTED_PLAYER_COUNT
```

Comment:
if you remove the optional flag you can handle this more sanely:
```suggestion
		this.expectedPlayerCount = opts.expectedPlayerCount ?? DEFAULT_EXPECTED_PLAYER_COUNT
```